src/main/resources/i18n/messages.properties

```diff
@@ -95,6 +95,8 @@ Dialog.Restart=%s needs to be restarted to apply the changes. Do you want to res
 Dialog.Select=Select
 Dialog.Warning=Warning
 Dialog.YES=Yes
+Dialog.DefaultsYes=Yes, run the wizard
```

Comment:
These new options should have Wizard keys instead of Dialog keys, as they are very specific to wizard functionality